libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -1114,27 +1060,68 @@ def _make_message_chunk_from_anthropic_event(
     event: anthropic.types.RawMessageStreamEvent,
     *,
     stream_usage: bool = True,
+    coerce_content_to_string: bool = False,
```

Comment:
why does this need a default value?

Reply:
If internal helper methods, might be more stable if we remove default, and has to be passed in. Don't feel super strongly though

Reply:
Sure